context/app/routes_main.py

```diff
@@ -83,6 +83,13 @@ def collection_details_ext(uuid, ext):
     return collection
 
 
+@blueprint.route('/browse/<type>/HBM<hbm_suffix>')
```

Comment:
Is there a way to do this without knowing the type? That would greatly simplify things for everyone.

Reply:
Sure... I think I set it up this way for consistency with the others, but not necessary. (If you provide the wrong one, it will still redirect.)